### DISCUSSION TITLE
Privatevpn configs have spaces in the names and i cannot load them

### DISCUSSION THREAD

**Discussion by inertia666** [haugene/vpn-configs-contrib#201] [Q&A] [answered]
I am trying to add a list of vpn configs using privatevpn but the majority of the ovpn files have spaces in their names.

- OPENVPN_CONFIG=NO Oslo

I have tried adding a backslash but it doesn't work. Neither does putting the name in the quotes.

is there a solution to this?

**Comment by pkishino** [comment]
you have tried :dev branch, right? I thought this was fixed a while ago..

**Reply by inertia666** [reply]
> you have tried :dev branch, right? I thought this was fixed a while ago..

No, i have tried the main branch and the beta branch to test Transmission 4, and it's the same problem.

This isn't a deal breaker yet because the vpn i use doesn't have spaces, but I cannot give suitable backup vpns as they have the spaces in the names. I'd prefer to avoid the dev branch tbh.

**Comment by pkishino** [comment, marked as answer]
main branch is very outdated…
For any issue we always ask users to test dev branch first

**Reply by inertia666** [reply]
Tested the dev branch and it loads the correct vpn with spaces in the name. I'll keep it on the dev branch in that case. Thanks very much.

**Comment by pkishino** [comment]
Great
